docker-compose.yml

```diff
@@ -13,16 +13,16 @@ services:
       - ../dump:/docker-entrypoint-initdb.d
       # for importing a dump
       # if you want to re-import simply put away ../mysql/*
-  solr:
-    image: solr:5.3
-    ports:
-      - "127.0.0.1:8984:8983"
-    volumes:
-      - solr:/opt/solr/server/solr/mycores
-    entrypoint:
-      - docker-entrypoint.sh
-      - solr-precreate
-      - default
+#  solr:
```

Comment:
But I'll add a separate docker-compose-production.yml for this, so we might as well reenable this here.